### DISCUSSION TITLE
[General][Java SDK (Component)] Add spring-security and oauth sample

### DISCUSSION THREAD

**Discussion by heliang666s** [apache/dubbo#14879] [General]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java Samples (apache/dubbo-samples)

### Details

Just an example to show how to support OAuth and Spring Securry in Dubbo REST.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Similarityoung** [comment]
I would like to take on this task. Could you please assign it to me?
